install/helm/agones/templates/crds/_fleetautoscalerpolicy.yaml

```diff
@@ -0,0 +1,237 @@
+        key:  # The name of the Counter.
+          type: string
+        minCapacity:  # Minimum aggregate counter capacity that can be provided by this FleetAutoscaler. If not specified, the actual minimum capacity will be bufferSize.
+          type: integer
+          minimum: 0
+        maxCapacity:  # Maximum aggregate counter capacity that can be provided by this FleetAutoscaler. Required.
+          type: integer
+          minimum: 1
+        bufferSize:  # Size of a buffer of counted items that are available in the Fleet (available capacity). It can be specified either in absolute (i.e. 5) or percentage format (i.e. 5%).
+          x-kubernetes-int-or-string: true
+          anyOf:
+            - type: integer
+            - type: string
+    list:
+      type: object
+      nullable: true
+      required:
+        - key
+        - bufferSize
+        - maxCapacity
+      properties:
+        key:  # The name of the List.
+          type: string
+        minCapacity:  # Minimum aggregate list capacity that can be provided by this FleetAutoscaler. If not specified, the actual minimum capacity will be bufferSize.
+          type: integer
+          minimum: 0
+        maxCapacity:  # Maximum aggregate list capacity that can be provided by this FleetAutoscaler. Required.
+          type: integer
+          minimum: 1
+        bufferSize:  # Size of a buffer based on the list capacity that is available over the current aggregate list length in the Fleet. It can be specified either in absolute (i.e. 5) or percentage format (i.e. 5%).
+          x-kubernetes-int-or-string: true
+          anyOf:
+            - type: integer
+            - type: string
+    {{- $featureGates := include "agones.featureGates" . | fromYaml }}
+    {{- if not $featureGates.ScheduledAutoscaler }}
+    chain:
+        type: array
+        nullable: true
+        items:
+          type: object
+          nullable: true
+          required:
+            - policy
+          properties:
+            uid: # The uid of a chain entry.
+              type: string
+            schedule: # Defines when the policy is applied.
+              type: object
+              properties:
+                timezone: # Timezone to be used for the between period and active period.
+                  type: string
+                between:
+                  type: object
+                  start: # Defines when to start evaluating the active period.
+                    type: string
+                  end: # Defines when to stop evaluating the active period.
+                    type: string
+                activePeriod:
+                  type: object
+                  startCron: # Cron expression defining when to start applying the policy.
+                    type: string
+                  duration: # The length of time the policy should be applied for.
+                    type: string
+            policy: # Defines which policy to apply during the active period. Required.
```

Comment:
Rather than copy-paste, let's make it a nested include.

There was a good example in https://github.com/googleforgames/agones/pull/3718 which didn't get merged that meant it didn't get recursive forever.